EventFilter/CTPPSRawToDigi/python/totemRawToDigi_cff.py

```diff
@@ -1,7 +1,18 @@
 import FWCore.ParameterSet.Config as cms
 
 from CondFormats.CTPPSReadoutObjects.TotemDAQMappingESSourceXML_cfi import *
-TotemDAQMappingESSourceXML.mappingFileNames.append("CondFormats/CTPPSReadoutObjects/xml/ctpps_210_mapping.xml")
+totemDAQMappingESSourceXML.configuration = cms.untracked.VPSet(
+  cms.PSet(
+    validityRange = cms.untracked.EventRange("1:min - 280385:max"),
+    mappingFileNames = cms.untracked.vstring("CondFormats/CTPPSReadoutObjects/xml/ctpps_mapping_to_fill_5288.xml"),
+    maskFileNames = cms.untracked.vstring()
+  ),
+  cms.PSet(
+    validityRange = cms.untracked.EventRange("281601:min - 999999999:max"),
```

Comment:
what will happen for runs 280386 to 281600?
I'd expect that the coverage should be continuous.